src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {
```

Comment:
The interface name 'AreaCalculable' is a noun, which is good practice for class names, but since interfaces often describe abilities or functionalities, they are usually named with adjectives. Consider renaming this interface to something like 'Calculable' or 'Measurable' to better represent its purpose.